packages/storybook-helpers/main.ts

```diff
@@ -32,7 +32,8 @@ const staticDocumentsRule = {
   },
 };
 
-const webStories = "../src/**/*.stories.@(js|jsx|ts|tsx|mdx)";
+const webStories =
+  "../src/**/AssessmentResultsTable.stories.@(js|jsx|ts|tsx|mdx)";
```

Comment:
A temporary thing I assume for testing, ought to reverse it now 😛

Reply:
Thanks I totally forgot! Fixed by [008c4d4](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9324/commits/008c4d4ef3348e3a032a5b825c4174151256d856)